### ISSUE TITLE
Why  dilate gtmask

### ISSUE THREAD

**Issue by surheaven** [jby1993/SelfReconCode#40]
mgtMs = torch.nn.functional.max_pool2d(gtMs, kernel_size=3, stride=1, padding=1)
Why use this dilation , won't this make the mask inaccurate?

**Comment by jby1993** [comment]
I remember that pytorch3d differentiable soft mask render will produce an influence radius in the border, So I empirically dilate the gt mask to match the soft border, otherwise, the optimized geometry is slightly scaled down.

**Comment by surheaven** [comment]
Thanks!
